components/__design__/list.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { CSSProperties } from 'react';
+
+const listContainer = {
+  padding: '1.5rem',
+} as CSSProperties;
+
+export const OrderedLists: StoryObj = {
+  render: () => {
+    return (
+      <div style={listContainer}>
+        <ol>
+          <li>Ordered lists</li>
+          <li>Like this</li>
+          <li>with numbers</li>
+        </ol>
+      </div>
+    );
+  },
+};
+
+export const UnorderedLists: StoryObj = {
+  render: () => {
+    return (
```

Comment:
No need of return statements and `{}` you can remove them :). (On the one above too)

Reply:
Fixed